src/System.Net.Sockets.Legacy/tests/FunctionalTests/SendReceive.cs

```diff
@@ -507,7 +507,7 @@ private static void SendRecvAPM_Stream_TCP(IPAddress listenAt, bool useMultipleB
                 sendHandler(0);
             });
 
-            Assert.True(serverFinished.Task.Wait(TestTimeout));
+            Assert.True(serverFinished.Task.Wait(TestTimeout), "Timed out");
```

Comment:
I feel like the description is backwards here.  It'll say something like

```
Timed out:
   Expected: True
   Actual: False
```

"Completed within allowed time", or something like that, would lead to a less confusing result if this trips.